demisto_sdk/commands/common/constants.py

```diff
@@ -1365,6 +1365,8 @@ class PB_Status:
     FileType.MODELING_RULE: "6.10.0",
     FileType.LAYOUT_RULE: "6.10.0",
 }
+
+DEFAULT_PYTHON_VERSION = "3.10"
```

Comment:
I saw one for PY2 somewhere, let's put them here together

Reply:
The PY2 version is used only for docker helper (if no image is there)